banksystem/accounts/views.py

```diff
@@ -1,10 +1,13 @@
-from django.shortcuts import render
+from django.views.generic import TemplateView
+
 from .models import Account
 
 
-def get_bank_accounts(request, bank_id):
-    user = request.user
-    accounts_data = Account.objects.filter(user=user, bank_id=bank_id)
-    total_accounts = accounts_data.count()
+class GetBankAccount(TemplateView):
```

Comment:
dont change the existing function based views. rather make these classes separate, and have separate urls for them